realm/realm-library/src/main/java/io/realm/internal/TableOrView.java

```diff
@@ -334,8 +334,12 @@
 
     String toString();
 
+    // For testing purpose only. It might crash in UTF16 conversion.
+    @Deprecated
     String toString(long maxRows);
```

Comment:
1. We might want use for debugging internally
2. Object store is coming
3. I am lazy ....